src/machines/java/com/enderio/machines/common/blockentity/VacuumChestBlockEntity.java

```diff
@@ -54,6 +47,11 @@ public void handleEntity(ItemEntity entity) {
         }
     }
 
+    @Override
+    public String getColour() {
```

Comment:
as mojang usually writes code with american english and other parts use it too
```suggestion
    public String getColor() {
```

Reply:
Also add the useonly annotation